src/test/java/com/powsybl/openloadflow/ac/DistributedSlackOnLoadTest.java

```diff
@@ -83,4 +88,21 @@ void testWithLoadDetail() {
         assertActivePowerEquals(12.727, l5.getTerminal());
         assertActivePowerEquals(-50, l6.getTerminal()); // same as p0 because p0 < 0
     }
+
+    @Test
+    void testPowerFactorConstant() {
+        // given
+        Network testNetwork = Importers.loadNetwork(Paths.get("src", "test", "resources", "2.xiidm"));
+        parameters.setBalanceType(LoadFlowParameters.BalanceType.PROPORTIONAL_TO_CONFORM_LOAD);
+
+        // when
+        parametersExt.setPowerFactorConstant(false);
+        LoadFlowResult loadFlowResult1 = loadFlowRunner.run(testNetwork, parameters);
+
+        parametersExt.setPowerFactorConstant(true);
+        LoadFlowResult loadFlowResult2 = loadFlowRunner.run(testNetwork, parameters);
+
+        // then
+        assertBetterResults(loadFlowResult1, loadFlowResult2);
```

Comment:
I am not sure to understand why a result should be better than another one. Here the idea is to assert the reactive power that has changed in the second run. The assert on LoadFlowResults could be interested if you give reference loadflow results to avoid regression.